colcon_bundle/task/python/bundle.py

```diff
@@ -42,12 +42,18 @@ def add_arguments(self, *, parser):  # noqa: D102
                 'version_lt': '<',
```

Comment:
On another note, `version_neq` is not defined in colcon-ros (see [here](https://github.com/colcon/colcon-ros/blob/a5e17754338daa8ed1d24e25f98f5da503c13d2a/colcon_ros/package_identification/ros.py#L200))

Reply:
The plan is to include all the version specifiers defined by PEP 440 (since having too many specifiers won't break the code), so we can stay up-to-date when (if?) they are added in colcon-core.